server/app/mutations/stacks/common.rb

```diff
@@ -14,9 +14,9 @@ def handle_service_outcome_errors(service_name, messages, type)
       end
     end
 
-    def handle_volume_outcome_errors(volume_name, messages)
-      messages.each do |key, msg|
-        add_error(:volumes, key.to_sym, "Volume create failed for volume '#{volume_name}': #{msg}")
+    def handle_volume_outcome_errors(volume_name, errors)
+      errors.each do |key, atom|
+         add_service_error(volume_name, key, atom.symbolic, atom.message)
```

Comment:
Hmm, #1976 isn't merged yet... and this should not be under the `services => ...` key?

```
add_error("volumes.#{volume_name}.#{key}", atom.symbolic, message)
```